tests/vmi_configuration_test.go

```diff
@@ -110,6 +110,64 @@ var _ = Describe("Configurations", func() {
 				Expect(err).ToNot(HaveOccurred())
 			})
 
+			FIt("should report 3 socket under guest OS", func() {
+				vmi.Spec.Domain.CPU = &v1.CPU{
+					Sockets: 3,
+				}
+				vmi.Spec.Domain.Resources = v1.ResourceRequirements{
+					Requests: kubev1.ResourceList{
+						kubev1.ResourceMemory: resource.MustParse("64M"),
+					},
+				}
+
+				By("Starting a VirtualMachineInstance")
+				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
Please add labels to Expects. Without the labels, the error, if any, will not tell much information, eg.
```
Expected err to be nil.
```
With label: `Expect(err).ToNot(HaveOccurred(), "should start vmi")` it will print the label along
```
should start vmi:
Expected err to be nil.
```

Reply:
done